lib/compress/zstdmt_compress.c

```diff
@@ -102,9 +102,8 @@ typedef struct ZSTDMT_bufferPool_s {
     buffer_t bTable[1];   /* variable size */
 } ZSTDMT_bufferPool;
 
-static ZSTDMT_bufferPool* ZSTDMT_createBufferPool(unsigned nbWorkers, ZSTD_customMem cMem)
+static ZSTDMT_bufferPool* ZSTDMT_createBufferPool(ZSTD_customMem cMem, unsigned maxNbBuffers)
```

Comment:
why changing the order of arguments ?

Reply:
Originally I just added maxNbBuffers to the end, and I only removed nbWorkers later after the compiler warned me it was unused. I updated the PR to keep the original order of arguments.